crates/indexer/src/api/mod.rs

```diff
@@ -9,7 +9,3 @@ pub mod handlers;
 pub mod middleware;
 /// Route definitions and router creation
 pub mod routes;
-
-// TODO: Remove unused import after development phase
-#[allow(unused_imports)]
-pub use routes::create_router;
```

Comment:
I think we can leave it

Reply:
https://github.com/NethermindEth/wei/pull/83/commits/5709e49b3689ca31542dd4e86f4fdfa30b294677